requirements-dev.txt

```diff
@@ -1,15 +1,15 @@
 -e .
-aioconsole==0.1.15
```

Comment:
Don't drop the pins entirely. If you want to have a separate requirement file with direct test deps, that's fine. But put the pins in a constraint file, then. It can be generated with pip-tools.

Reply:
Thanks for reminding me to do this, I disabled pins for testing purposes. Willfix.